src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+spring.application.name=rachadinha
+
+spring.jpa.hibernate.ddl-auto=update
+logging.level.org.springframework.jdbc.core=DEBUG
+
+
+springdoc.api-docs.path=/api-docs
+springdoc.swagger-ui.path=/swagger-ui-custom.html
+springdoc.swagger-ui.operationsSorter=method
+
+spring.datasource.url=jdbc:mysql://rachadinha-server.mysql.database.azure.com:3306/rachadinhadb
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.datasource.username=user_rachadinha
+spring.datasource.password=!IfsulAgile
```

Comment:
Please remove this password, this should only be accessible through environment variables

Reply:
Take a look at [this article](https://www.baeldung.com/spring-boot-properties-env-variables)